src/bidiMapper/modules/cdp/CdpTargetManager.ts

```diff
@@ -149,20 +151,42 @@ export class CdpTargetManager {
     parentSessionCdpClient: CdpClient
   ) {
     const {sessionId, targetInfo} = params;
+
+    if (
+      this.#selfTargetId !== targetInfo.targetId &&
+      targetInfo.type !== 'service_worker'
+    ) {
+      // Service workers are special case because they attach to the
```

Comment:
Left over comment?

Reply:
no, it still applies to the service worker check

Reply:
I have updated it again to fix WPT